bindings/rust/s2n-tls/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "s2n-tls"
 description = "A C99 implementation of the TLS/SSL protocols"
-version = "0.0.19"
+version = "0.1.0"
 authors = ["AWS s2n"]
```

Comment:
Why the minor version bump? For my own edification, what functionality/APIs are we adding?

Reply:
I did a minor bump for #3523 and #3642

Reply:
I suppose CRL is unstable so that doesn't count. I can do a patch version if you think thats better, it wasn't really clear to me what was warranted

Reply:
CRL didnt change the bindings code though. Also the Deref change is forwards compatible so patch should be ok i think.

Reply:
if CRL wasn't unstable, wouldn't it automatically change the s2n-tls-sys bindings and warrant a minor change? Or are we only doing minor bumps for mid level bindings changes?

Reply:
Since these crates are pre 1.0 I dont know if we have a great story around this.

I tend to treat patch as non-breaking changes and minor as breaking changes for the pre-1.0 crate. The sys crate will never be stable because its unstable apis and customers should never have to use it/rely on it.